### ISSUE TITLE
Bower doesn't mark project private when answered yes in bower init prompt

### ISSUE THREAD

**Issue by remcohaszing** [bower/bower#1522]
When `bower init` promps whether or not to mark a package as private and 'yes' is answered, it is still not marked private.

The printed json is the same as is written to the `bower.json` file.

```sh
$ bower --version
1.3.10
$ bower init
? name: bar
? version: 0.0.0
? description: 
? main file: 
? what types of modules does this package expose?: 
? keywords: 
? authors: Remco Haszing <remcohaszing@gmail.com>
? license: MIT
? homepage: 
? set currently installed components as dependencies?: Yes
? add commonly ignored files to ignore list?: Yes
? would you like to mark this package as private which prevents it from being accidentally published to the registry?: Yes

{
  name: 'bar',
  version: '0.0.0',
  authors: [
    'Remco Haszing <remcohaszing@gmail.com>'
  ],
  license: 'MIT',
  ignore: [
    '**/.*',
    'node_modules',
    'bower_components',
    'test',
    'tests'
  ]
}

? Looks good?: (Y/n) 
```

**Comment by Parakleta** [comment]
This issue was resolved by commit 3df6144 but then regressed in commit 45bab9f.  The problem is caused by https://github.com/mout/mout/commit/d80a75510db77fa11e0b6cc795a7d1f432892457.

**Comment by felixrabe** [comment]
+1, this is not fixed in 1.4.1 because of the regression. Please reopen.

**Comment by albertinad** [comment]
@remcohaszing , @sheerun  I've tested it with latest version available, 1.6.5, and I wasn't able to reproduce it. It seems to be fixed and working as expected. @remcohaszing can you try it with the latest version? 
Thanks!

**Comment by Parakleta** [comment]
This appears to have been fixed in 304b6393d424b8f886a599a40e8cc0c28fa2d1a8

**Comment by albertinad** [comment]
Thanks @Parakleta ! I will close this issue since it was fixed. If this happens again, feel free to re-open it.
